lib/action_cable/subscription_adapter/solid_cable.rb

```diff
@@ -104,8 +104,18 @@ def broadcast_messages
             end
 
             def with_polling_volume
-              if ::SolidCable.silence_polling?
-                ActiveRecord::Base.logger.silence { yield }
+              if ::SolidCable.silence_polling? && ActiveRecord::Base.logger
+                if ActiveRecord::Base.logger.respond_to?(:silence)
```

Comment:
I was getting an error, as silence is not present in rails application (7.2.1):
```
'with_polling_volume': undefined method `silence' for an instance of Logger (NoMethodError)
ActiveRecord::Base.logger.silence { yield }
```

Reply:
If this PR is merged i will do a PR to fix silence also for those who want to use solid_queue in earlier version that Rails 8